front/screens/aroundMe/tabAroundMeScreen.component.tsx

```diff
@@ -216,12 +232,27 @@ const TabAroundMeScreen: React.FC = () => {
         }}
         showSnackBarWithMessage={showSnackBarWithMessage}
         setIsLoading={setIsLoading}
-        updateUserLocation={(coordinates: LatLng) => {
-          setSelectedPoiIndex(-1);
-          setCurrentUserLocation(coordinates);
-          moveMapToCoordinates(coordinates.latitude, coordinates.longitude);
+        updateUserLocation={async (coordinates: LatLng | undefined) => {
+          if (coordinates) {
+            setSelectedPoiIndex(-1);
+            setCurrentUserLocation(coordinates);
+            moveMapToCoordinates(coordinates.latitude, coordinates.longitude);
+          } else {
+            const savedRegion: Region | undefined = await StorageUtils.getObjectValue(
+                StorageKeysConstants.cartoSavedRegion
+              );
+            moveMapToCoordinates(
+              savedRegion?.latitude
```

Comment:
J'imagine que quelque chose comme marche :

```suggestion
              savedRegion?.latitude || AroundMeConstants.COORDINATE_PARIS.latitude
```